www/pfSense-pkg-squid/files/usr/local/pkg/squid_antivirus.inc

```diff
@@ -705,13 +705,17 @@ function squid_write_cicap_rcfile() {
 	$cicap_start_cmd = "{$cicap_bin} -f {$cicap_conf}";
 	$cicap_stop_cmd = <<< EOF
 
-if [ -p "{$cicap_pipe}" ]; then
-	/bin/echo -n "stop" > /var/run/c-icap/c-icap.ctl
-else
-	/usr/bin/killall -9 c-icap 2>/dev/null
+if /bin/pgrep -aq c-icap 2>/dev/null; then
+	if [ -p "{$cicap_pipe}" ]; then
+		/bin/echo -n "stop" > /var/run/c-icap/c-icap.ctl
```

Comment:
Now that I'm looking closer, couldn't this also be:  `/bin/echo -n "stop" > {$cicap_pipe}`?

Reply:
Probably, been this way for ages, not sure why... 👍

Reply:
Indeed. Not a big deal though. Fix if you want, or let it go.